activemodel/lib/active_model/railtie.rb

```diff
@@ -16,5 +17,10 @@ class Railtie < Rails::Railtie # :nodoc:
     initializer "active_model.i18n_customize_full_message" do
       ActiveModel::Error.i18n_customize_full_message = config.active_model.delete(:i18n_customize_full_message) || false
     end
+
+    initializer "active_model.enforce_iso_date_strings" do
+      ActiveModel::Type::Date.enforce_iso_date_strings = config.active_model.enforce_iso_date_strings
+      ActiveModel::Type::DateTime.enforce_iso_date_strings = config.active_model.enforce_iso_date_strings
```

Comment:
I passed the property via class attributes to avoid making the types dependent on the Rails application config. I saw we do this with `ActiveModel::Error.i18n_customize_full_message`, but I am not sure if there is a better convention here.